app/frontend/src/styles/application/publishers/publisher.scss

```diff
@@ -1,21 +1,3 @@
-$desktop-container-width: 1200px;
-
-.publisher {
-  .desktop-container-width {
-    @include govuk-media-query($from: tablet) {
-      width: $desktop-container-width;
-    }
-  }
-
-  .vacancy-actions {
-    .govuk-button {
-      @include govuk-media-query($from: desktop) {
-        margin-right: govuk-spacing(2);
-      }
-    }
-  }
-
-  .job-application-actions {
-    display: flex;
-  }
+.job-application-actions {
```

Comment:
wondering if this could dissappear also and where its used replaced with `govuk-button-group` also?